<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryBigWidget</class>
 <widget class="QWidget" name="CategoryBigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CategoryBigListView" name="listView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView{
border:none;
background-color: rgb(81, 80, 80);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="lastPageButton">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-image: url(:/setImage/left.png);
}

QPushButton:pressed{
border-image: url(:/setImage/left_press.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #383838;
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPageButton">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>18</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-image: url(:/setImage/right.png);
}

QPushButton:pressed{
border-image: url(:/setImage/right_press.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CategoryBigListView</class>
   <extends>QListView</extends>
   <header>categorybiglistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
